app/controllers/broker_agencies/profiles_controller.rb

```diff
@@ -182,6 +183,42 @@ def family_index
     end
   end
 
+  def commission_statements
+    permitted = params.permit(:id)
+    @id = permitted[:id]
+    if current_user.has_broker_role?
+      @broker_agency_profile = BrokerAgencyProfile.find(current_user.person.broker_role.broker_agency_profile_id)
+    elsif current_user.has_hbx_staff_role?
+      @broker_agency_profile = BrokerAgencyProfile.find(BSON::ObjectId.from_string(@id))
+    else
+      redirect_to new_broker_agencies_profile_path
+      return
+    end
+    documents = @broker_agency_profile.organization.documents
+    if documents
+      @statements = get_commission_statements(documents)
+    end
+    collect_and_sort_commission_statments
```

Comment:
minor typo on method name collect_and_sort_comission_statments collect_and_sort_comission_statements

Reply:
fixed here and had to make the same typo fix in the method definition